app/components/project-geometries/modes/draw.js

```diff
@@ -44,6 +56,14 @@ export default class DrawComponent extends Component {
     const [selectedId] = draw.getSelectedIds();
 
     if (firstSelectedFeature) {
+      /* Bring properties.missingLabel up to top-level property
+       * Must be top-level for watching in the feature-label-form,
+       * but cannot be added as top-level in the overriden mapbox-gl-draw mode event
+       * (See https://github.com/NYCPlanning/labs-applicantmaps/issues/417)
+       */
+      if ('missingLabel' in firstSelectedFeature.properties) {
+        firstSelectedFeature.missingLabel = firstSelectedFeature.properties.missingLabel;
+      }
```

Comment:
Hmm was the point of this so that it becomes observable? Because you could not observe a nested leaf on an `@each` in a computed?